setup.py

```diff
@@ -219,6 +219,17 @@ def run(self):
                 raise RuntimeError(
                     "-DONNX_DISABLE_EXCEPTIONS=ON option is only available for c++ builds. Python binding require exceptions to be enabled."
                 )
+            if "PYTHONPATH" in os.environ:
+                # To solve problem that the extra PYTHONPATH env variable added by pip.
+                segs = os.environ["PYTHONPATH"].split(":")
```

Comment:
This won't work on Windows.

Reply:
@xadupre Does windows has the same problem? Is it OK to skip windows platform using `"PYTHONPATH" in os.environ and sys.platform != "win32"`

Reply:
I assume miniconda works the same way on Windows. You remove every path from PYTHONPATH already added in sys.path so it starts looking cmake in the installed packages first. Is it possible to add a longer comment then to tell this is needed for conda.

Reply:
@xadupre I can get the same error in Windows env. With the new fix, the problem is also fixed in Windows env.
![image](https://user-images.githubusercontent.com/19584326/236001784-b6770ec2-252d-40f8-939f-7e4f84ce6a94.png)

With the fix:

![image](https://user-images.githubusercontent.com/19584326/236001823-78bb1361-09f5-4741-bf8b-fbb312841eb5.png)